packages/client/src/arpa_reporter/views/UserView.vue

```diff
@@ -1,6 +1,6 @@
 <template>
   <div>
-    <h2>User</h2>
+    <h1>User</h1>
```

Comment:
Note to reviewer:  before/after screenshots

## Before
![image](https://github.com/user-attachments/assets/91a9cdf3-4aa8-4833-b574-687020a0a74d)


## After
![image](https://github.com/user-attachments/assets/090c2008-cd65-44e0-9e44-a6bd746c06b0)